### ISSUE TITLE
Cocoapods doesn't understand Xcode projects that also use Swift Package Manager

### ISSUE THREAD

**Issue by 0xTim** [CocoaPods/CocoaPods#8885]
TL;DR:
Cocoapods doesn't understand new stuff in Xcode 11 since it's only just been released 😄 

When running `pod install` on a workspace that contains a project that contains Swift Package Manager dependencies as well it fails. It should be able to use both dependency managers side by side

### Command

```
/Users/timc/.rbenv/versions/2.5.3/bin/pod install
```

### Report

* What did you do?
Ran `pod install` on a workspace that contains SPM projects as well
* What did you expect to happen?
`pod install` to execute successfully
* What happened instead?
`pod install` failed (see below)

### Stack

```
   CocoaPods : 1.7.1
        Ruby : ruby 2.5.3p105 (2018-10-18 revision 65156) [x86_64-darwin18]
    RubyGems : 2.7.6
        Host : Mac OS X 10.14.5 (18F132)
       Xcode : 10.2.1 (10E1001)
         Git : git version 2.20.1 (Apple Git-117)
Ruby lib dir : /Users/timc/.rbenv/versions/2.5.3/lib
Repositories : bbc - git@github.com:bbc/map-ios-podspecs.git @ bc801c78b82f56dd9e21a095de546814137607a3
               master - https://github.com/CocoaPods/Specs.git @ 845cc138fa183f4dda28ca9ff173b419b6d66623
```

### Plugins

```
cocoapods-deintegrate : 1.0.3
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.0
cocoapods-stats       : 1.1.0
cocoapods-trunk       : 1.3.1
cocoapods-try         : 1.1.0
```

### Podfile

```ruby
platform :ios, '11.0'

target 'IceCreamShop' do
  use_frameworks!
  pod 'SnapKit', '~> 5.0.0'
end
```

### Error

```
RuntimeError - [Xcodeproj] Unknown object version.
/Users/timc/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/xcodeproj-1.9.0/lib/xcodeproj/project.rb:227:in `initialize_from_file'
/Users/timc/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/xcodeproj-1.9.0/lib/xcodeproj/project.rb:112:in `open'
/Users/timc/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/cocoapods-1.7.1/lib/cocoapods/installer/analyzer.rb:1051:in `block (2 levels) in inspect_targets_to_integrate'
/Users/timc/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/cocoapods-1.7.1/lib/cocoapods/installer/analyzer.rb:1050:in `each'
/Users/timc/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/cocoapods-1.7.1/lib/cocoapods/installer/analyzer.rb:1050:in `block in inspect_targets_to_integrate'
/Users/timc/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/cocoapods-1.7.1/lib/cocoapods/user_interface.rb:64:in `section'
/Users/timc/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/cocoapods-1.7.1/lib/cocoapods/installer/analyzer.rb:1045:in `inspect_targets_to_integrate'
/Users/timc/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/cocoapods-1.7.1/lib/cocoapods/installer/analyzer.rb:100:in `analyze'
/Users/timc/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/cocoapods-1.7.1/lib/cocoapods/installer.rb:398:in `analyze'
/Users/timc/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/cocoapods-1.7.1/lib/cocoapods/installer.rb:221:in `block in resolve_dependencies'
/Users/timc/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/cocoapods-1.7.1/lib/cocoapods/user_interface.rb:64:in `section'
/Users/timc/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/cocoapods-1.7.1/lib/cocoapods/installer.rb:220:in `resolve_dependencies'
/Users/timc/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/cocoapods-1.7.1/lib/cocoapods/installer.rb:156:in `install!'
/Users/timc/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/cocoapods-1.7.1/lib/cocoapods/command/install.rb:51:in `run'
/Users/timc/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/claide-1.0.2/lib/claide/command.rb:334:in `run'
/Users/timc/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/cocoapods-1.7.1/lib/cocoapods/command.rb:52:in `run'
/Users/timc/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/cocoapods-1.7.1/bin/pod:55:in `<top (required)>'
/Users/timc/.rbenv/versions/2.5.3/bin/pod:23:in `load'
/Users/timc/.rbenv/versions/2.5.3/bin/pod:23:in `<main>'
```

――― TEMPLATE END ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――

[!] Oh no, an error occurred.

Search for existing GitHub issues similar to yours:
https://github.com/CocoaPods/CocoaPods/search?q=%5BXcodeproj%5D+Unknown+object+version.&type=Issues

If none exists, create a ticket, with the template displayed above, on:
https://github.com/CocoaPods/CocoaPods/issues/new

Be sure to first read the contributing guide for details on how to properly submit a ticket:
https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md

Don't forget to anonymize any private data!

Looking for related issues on cocoapods/cocoapods...
 - Xcode 11 Beta Crash
   https://github.com/CocoaPods/CocoaPods/issues/8871 [open] [3 comments]
   2 days ago

 - Xcodeproj doesn't know about the following attributes {"inputFileListPaths"=>[], "outputFileListPaths"=>[]} for the 'PBXShellScriptBuildPhase' isa
   https://github.com/CocoaPods/CocoaPods/issues/7835 [closed] [49 comments]
   19 hours ago

 - - ERROR | [iOS] unknown: Encountered an unknown error (Malformed version number string ) during validation.
   https://github.com/CocoaPods/CocoaPods/issues/8240 [closed] [19 comments]
   2 weeks ago

and 45 more at:
https://github.com/cocoapods/cocoapods/search?q=%5BXcodeproj%5D%20Unknown%20object%20version.&type=Issues&utf8=✓

[!] [!] Xcodeproj doesn't know about the following attributes {"productRef"=>"23AF477322A9BD9000EDF2DA"} for the 'PBXBuildFile' isa.
If this attribute was generated by Xcode please file an issue: https://github.com/CocoaPods/Xcodeproj/issues/new

[!] [!] Xcodeproj doesn't know about the following attributes {"packageProductDependencies"=>["23AF477322A9BD9000EDF2DA"]} for the 'PBXNativeTarget' isa.
If this attribute was generated by Xcode please file an issue: https://github.com/CocoaPods/Xcodeproj/issues/new

[!] [!] Xcodeproj doesn't know about the following attributes {"packageReferences"=>["23AF477222A9BD9000EDF2DA"]} for the 'PBXProject' isa.
If this attribute was generated by Xcode please file an issue: https://github.com/CocoaPods/Xcodeproj/issues/new

**Comment by dnkoutso** [comment]
`gem update fourflusher` to fix the crash.

PR to update Swift Package Manager and Xcode 11 objects is here. https://github.com/CocoaPods/Xcodeproj/pull/687

Going to close as there is already another issue for this and the crash has been fixed.

**Comment by 0xTim** [comment]
👍 
FYI I'm still getting a crash after updating fourflusher, but will wait til the linked issue is closed until raising anything further

**Comment by dnkoutso** [comment]
actually apologies. The crasher is fixed by the PR I mentioned.

Fourflusher fixes `pod lib lint` with Xcode 11 simulator.

Still keeping it close as there is still an issue for Xcode 11 support.

**Comment by Umar-M-Haroon** [comment]
I'm having the same error even with Xcodeproj being set to the 'xcode11_support' branch in my Gemfile.
